### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by Tombo-tv**
Adding homework 3. Ready to review.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4861991&bid=16127260)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+            Console.WriteLine("Judging by our calculations, we have come to the conclusion that both users are not within the healthy range " +
+                              "\nof the self-confidence crusher BMI Calculator.");
+
+            Console.WriteLine("\nHave a nice day!");
+            Console.ReadLine();
+        }
+
+
+        public static void Person()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Please rename the function to be a verb and not a noun. 
HandlePerson, ProcessPerson sounds better ;)

Comment (ellmaster) [reply Src/BootCamp.Chapter/Lesson3.cs]: In addition:
We will make use of Person() (or some other nouns) later on. We don't want to mix functionnames with what comes then! :)
That is why this is important.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4861991&bid=16127260)

**Pull request closed without merging**